docs/modelzoo_statistics.md

```diff
@@ -0,0 +1,161 @@
+
+# Model Zoo
```

Comment:
The file should not be traced.
`modelzoo_statistics.md` will be generated by `stat.py` which is called in `conf.py`.

Reply:
The file still seems to be tracked.